config/locales/clearance.fr.yml

```diff
@@ -0,0 +1,56 @@
+---
+fr:
+  clearance:
+    models:
+      clearance_mailer:
+        change_password: "Modifier votre mot de passe"
+  clearance_mailer:
+    change_password:
+      change_my_password: "Modifier mon mot de passe"
+      closing: "Si vous n'êtes pas à l'origine de cette demande, ignorez ce courriel. Votre mot de passe n'a pas été changé."
+      link_text: "Changer mon mot de passe"
+      opening: "Quelqu'un (nous espérons que c'est vous) a demandé que nous vous envoyions un lien pour changer votre mot de passe :"
+  flashes:
+    failure_after_create: "Mauvais courriel ou mot de passe."
+    failure_after_update: "Le mot de passe ne peut pas être vide."
+    failure_when_forbidden: "Veuillez vérifier l'URL ou essayer de soumettre
+      le formulaire à nouveau."
+    failure_when_not_signed_in: "Veuillez vous connecter pour continuer."
+  helpers:
+    label:
+      password:
+        email: Adresse courriel
+      password_reset:
+        password: Choisir un mot de passe
+    submit:
+      password:
+        submit: Réinitialiser le mot de passe
+      password_reset:
+        submit: Sauvegarder le mot de passe
+      session:
+        submit: Se connecter
+      user:
+        create: Créez votre compte
+  layouts:
+    application:
+      sign_in: Se connecter
+      sign_out: Se déconnecter
+  passwords:
+    create:
+      description: "Vous recevrez un courriel dans les prochaines minutes. Il contient des instructions pour changer votre mot de passe."
+    edit:
+      description: "Votre mot de passe a été réinitialisé. Choisissez un nouveau mot de passe ci-dessous."
+      title: Modifier votre mot de passe
+    new:
+      description: Pour recevoir un courriel contenant un lien pour de réinitialisation de votre mot de passe, veuillez entrer votre adresse courriel.
```

Comment:
I think this should be, “un lien pour la réinitialisation” :)

Reply:
I've changed for "un lien de réinitialisation", thanks. If you see anything else, feel free to directly [suggest changes](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) in Github.

Reply:
Wow, cool, I don't think I've ever used the “suggesting changes” feature in GitHub. Thanks for the link and for making that change so quickly. 🙌